### PR TITLE
Minor improvements for metadata tutorial

### PR DESCRIPTION
**Pull request by cbrnr**
I mostly fixed some minor typos and reworded some sentences slightly.

### CODE REVIEW

**Comment by larsoner** [comment]
> Does it happen for you locally? Can you figure out why and fix if needed?

Without having looked at the code or the output, I generally find nowadays that problems like this are easily/magically solved with a `constrained_layout=True` in fig creation or `fig.set_layout_engine('constrained')` (along with stuff like padding being better)

File: tutorials/epochs/40_autogenerate_metadata.py [hunk 1]
```diff
@@ -157,12 +157,13 @@
...
 # column, ``response``, containing the latency of the respective event. This
-# value pertains only the first (or, in this specific example: the only)
+# value represents the first (or, in this specific example: the only)
 # response, regardless of side (left or right). To indicate **which** event
-# type (here: response side) was matched, a second column is added:
-# ``first_response``. The values in this column are the event types without the
-# string used for matching, as it is already encoded as the column name, i.e.
-# in our example, we expect it to only contain ``'left'`` and ``'right'``.
+# type (here: response side) was matched, a second column named
+# ``first_response``is added. The values in this column are the event types
```
Comment (agramfort) [inline tutorials/epochs/40_autogenerate_metadata.py:163]:
```suggestion
# ``first_response`` is added. The values in this column are the event types
```

**Review by agramfort — APPROVED** [review]

**Comment by cbrnr** [comment]
I removed the tight layout because of a `UserWarning`. I've now set a constrained layout, let's see if this works.

**Comment by cbrnr** [comment]
Yep, looks good now.

**Comment by larsoner** [comment]
Thanks @cbrnr !

**Pull request merged**